source/server/config_validation/server.h

```diff
@@ -89,6 +89,10 @@ class ValidationInstance : Logger::Loggable<Logger::Id::main>,
   ThreadLocal::Instance& threadLocal() override { return thread_local_; }
   const LocalInfo::LocalInfo& localInfo() override { return *local_info_; }
 
+  const std::chrono::milliseconds& statsFlushInterval() override {
```

Comment:
const  method per discussion in outdated conversation.  You will have to fix the underlying interface and impl.

not const ref return though.

Reply:
Sure, I should have paid attention. will correct all these.